### ISSUE TITLE
Windows 1903 Action center key triggers launch of Office App

### ISSUE THREAD

**Issue by Pixelpunker** [stephtr/FixSurfaceKeyboard#5]
Since manually upgrading to Windows 10 1903 (May 2019 Update, and dare I say, I will never upgrade prematurely again) using the action center key on the keyboard triggers the annoying launch of the Office App for Windows 10, even after "uninstalling" the Office App it reappears when using this key.
Very strange.
When setting the keyboard input language to english, this does not occur. It only occurs when setting the keyboard input language to german.
![Office App Launch 2](https://user-images.githubusercontent.com/20599815/59554065-2b906980-8f9e-11e9-9492-275062888469.png)

**Comment by stephtr** [comment]
That was a strange issue, but it's now solved by changing the way the app opens the action center (now it executes `ms-actioncenter:` instead of triggering the keyboard combination <kbd>Win</kbd> + <kbd>A</kbd>).
The updated app is available through [release 1.0.5](https://github.com/stephtr/FixSurfaceKeyboard/releases/tag/1.0.5).

**Comment by Pixelpunker** [comment]
Thank you for such a prompt response! Unfortunately, the issue still persists with version 1.0.5 on my machine. BTW hitting Win + A or executing ms-actioncenter: in the file explorer does not trigger the issue.

**Comment by stephtr** [comment]
Could you give [1.0.6](https://github.com/stephtr/FixSurfaceKeyboard/releases/tag/1.0.6) a try?

**Comment by stephtr** [comment]
The underlying reason for this bug might be, that the old version of the program together with the wrongly mapped action center key in the German keyboard layout triggers a so called Office key: https://twitter.com/h0x0d/status/1140784669962584066?s=09

**Comment by Pixelpunker** [comment]
> 
> 
> Could you give [1.0.6](https://github.com/stephtr/FixSurfaceKeyboard/releases/tag/1.0.6) a try?

I tried version 1.0.6 and the Office App no longer opens. Good! However, the key no longer functions as a toggle. That is it opens Action Center, but using the key again will not close it afterwards. This is different to how Windows+A works.

**Comment by stephtr** [comment]
tl;dr: It is finally fixed in [1.0.7](https://github.com/stephtr/FixSurfaceKeyboard/releases/tag/1.0.7).

The reason for the behavior of 1.0.6 was that I switched to just executing `ms-actioncenter:` which seemed to be more reliable (but one-way).
However I just took a further look and discovered, that since Windows 10 v1903 <kbd>Shift</kbd>+<kbd>Ctrl</kbd>+<kbd>Win</kbd>+<kbd>Alt</kbd> is the "Office key" (which [can open different Office apps](https://twitter.com/TrStephan/status/1142399137645772801)). This shortcut collided with <kbd>Shift</kbd>+<kbd>Ctrl</kbd>+<kbd>Win</kbd>+<kbd>Alt</kbd>+<kbd>F21</kbd>, which gets triggered by the surface keyboard when pressing the action center key. For 1.0.7 I rearranged some logic in the app to prevent this.
